app/src/main/java/com/example/mykotlincalculator21/mvp/model/CalculatorModel.kt

```diff
@@ -1,16 +1,59 @@
 package com.example.mykotlincalculator21.mvp.model
 
 import com.example.mykotlincalculator21.mvp.contract.CalculatorContract
-import com.example.mykotlincalculator21.utils.EMPTY_STRING
+import com.example.mykotlincalculator21.utils.*
 
 class CalculatorModel : CalculatorContract.CalculatorModelContract {
     private var firstOperand: String = EMPTY_STRING
+    private var secondOperand: String = EMPTY_STRING
+    private var operator: String = EMPTY_STRING
+    private var result: String = EMPTY_STRING
 
     override fun saveNumber(number: String) {
-        firstOperand = number
+        if (operator.isEmpty()) {
+            firstOperand += number
+        } else {
+            secondOperand += number
+        }
     }
 
-    override fun getResult(): String {
-        return firstOperand
+    override fun getPartialResult(): String {
+        return (firstOperand + operator + secondOperand)
```

Comment:
Here is better if you do this 
override fun getPartialResult() = "$firstOperand$operator$secondOperand"
Its a better form when we concat strings

Reply:
you can ommit the return word too:

fun getPartialResult() = $firstOper...

Reply:
Thx for the tips guys